yaml2rst.py

```diff
@@ -101,6 +101,9 @@ def convert(lines):
         elif line.startswith('# ') or line == '#':
             if state != STATE_TEXT:
                 yield ''
+            # Filter out [[[\d and ]]] at the end of a documentation line.
+            # Those sequences can be used for folding sections.
+            line = re.sub(r"\s*(:?\[{3}|\]{3})\d?$", "", line)
```

Comment:
>> This would strip foldmarks in every line, not only in rst-comments

>It should only affect text lines. Can you double check?

I thought it should only effect comment lines (in rst :-). As of now it effects all text lines, including code lines. E.g. this will remove the fold mark, too:

```yaml
# Example for using fold.marks::
#
#    Some Section [[[
#    --------------------------------
```

But maybe this is not a problem. Lets try it.

>>   \s* is of no use?

> It will additionally strip any trailing whitespace from the output file. Trailing whitespace is considered bad practice.

IC. For our usecase trailing whitespace is no problem: It's just in memory and rst does not care.. But anyway, we can keep te `\s*` in.

> >  What should the trailing digit be? (Please document)

> I documented that using an example in:

I can not find it there. Please add a short comment (e.g \d are the optional folding levels)